### ISSUE TITLE
Could NOT find CImg (missing: CIMG_INCLUDE_DIR CImg_FOUND)

### ISSUE THREAD

**Issue by certik** [jkriege2/JKQtPlotter#22]
After #21 is in, there is another issue on Linux (Ubuntu 18.04) that was not there before:
```
-- CImg: trying to locate CImg library
-- Could NOT find CImg (missing: CIMG_INCLUDE_DIR CImg_FOUND) 
-- ..   Building Example rgbimageplot_qt
-- ..   Building Example simpletest
-- ..   Building Example speed
-- ..   Building Example stackedbars
-- ..   Building Example stepplots
-- ..   Building Example styledboxplot
-- ..   Building Example styling
-- ..   Building Example symbols_and_errors
-- ..   Building Example symbols_and_styles
-- ..   Building Example ui
-- ..   Building Example user_interaction
-- ..   Building Example violinplot
-- Configuring incomplete, errors occurred!
See also "/home/certik/repos/JKQtPlotter/build/CMakeFiles/CMakeOutput.log".
```

**Comment by jkriege2** [comment]
Hi!

that is normal and should only affect the examples ... could you post the complete CMake output?

**Comment by certik** [comment]
Indeed, this does not cause CMake to fail. It only looked like this was the failure. The reason CMake failed for me was due to the reason explained in #21. And I provided a fix in #24. 

There is no problem regarding CImg.
